sql/core/src/main/scala/org/apache/spark/sql/execution/ExplainUtils.scala

```diff
@@ -218,6 +218,8 @@ object ExplainUtils extends AdaptiveSparkPlanHelper {
       plan: => QueryPlan[_],
       subqueries: ArrayBuffer[(SparkPlan, Expression, BaseSubqueryExec)]): Unit = {
     plan.foreach {
+      case adp: AdaptiveSparkPlanExec =>
```

Comment:
nit. It seems that we prefer `p` or `a` instead of `adp`.
```scala
$ git grep ': AdaptiveSparkPlanExec'
sql/core/src/main/scala/org/apache/spark/sql/execution/ExplainUtils.scala:          case p: AdaptiveSparkPlanExec =>
sql/core/src/main/scala/org/apache/spark/sql/execution/ExplainUtils.scala:      case p: AdaptiveSparkPlanExec => setCodegenId(p, Seq(p.executedPlan))
sql/core/src/main/scala/org/apache/spark/sql/execution/ExplainUtils.scala:      case p: AdaptiveSparkPlanExec => remove(p, Seq(p.executedPlan))
sql/core/src/main/scala/org/apache/spark/sql/execution/SparkPlanInfo.scala:      case a: AdaptiveSparkPlanExec => a.executedPlan :: Nil
sql/core/src/main/scala/org/apache/spark/sql/execution/adaptive/AdaptiveSparkPlanHelper.scala:    case a: AdaptiveSparkPlanExec => Seq(a.executedPlan)
sql/core/src/main/scala/org/apache/spark/sql/execution/adaptive/AdaptiveSparkPlanHelper.scala:    case a: AdaptiveSparkPlanExec => a.executedPlan
sql/core/src/test/scala/org/apache/spark/sql/SparkSessionExtensionSuite.scala:      case a: AdaptiveSparkPlanExec =>
```

Reply:
`p` is already used so I'll choose `a`.